python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -34,12 +34,12 @@
 # .. math::
 #
 #   h(\boldsymbol{\xi}) = \sum_{\boldsymbol{\alpha} \in \mathbb{N}^p}
-#   a_{\boldsymbol{\alpha}} \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi}) + \epsilon
+#   a_{\boldsymbol{\alpha}} \psi_{\boldsymbol{\alpha}}(\boldsymbol{\xi})
```

Comment:
Very true. Indeed (assuming the orthogonal basis of polynomials is complete), the expansion converges to the function.

Reply:
Shouldn't the fact the orthogonal basis of polynomials must be complete be added to the text?